src/DoubleTheDonation/FormExtension/Actions/InsertDefaultBlock.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace GiveDoubleTheDonation\DoubleTheDonation\FormExtension\Actions;
+
+use Give\DonationForms\Models\DonationForm;
+use Give\Framework\Blocks\BlockModel;
+
+/**
+ * @unreleased
+ */
+class InsertDefaultBlock
+{
+    public function __invoke(DonationForm $form)
+    {
+        $blockData = json_decode(
```

Comment:
Woah, nice use of the block.json file! Can we please write a unit test for this?

Reply:
Sure